src/snowcli/cli/main/snow_cli_main_typer.py

```diff
@@ -0,0 +1,31 @@
+import click
+from typer import Typer
+
+from snowcli.cli import DEFAULT_CONTEXT_SETTINGS, OutsideTyperGlobalContext
+
+
+class SnowCliMainTyper(Typer):
```

Comment:
Do we plan to have other than `Main` typers?

Reply:
I imagine we can have one main typer (entrypoint) and other typers (added by typer.add_typer) for purpose of command groups.

Main typer may be responsible for global exception handling (like here) but also for top-level settings (like these in the constructor).